src/mame/drivers/leapfrog_leappad.cpp

```diff
@@ -135,6 +136,22 @@ void leapfrog_leappad_state::leapfrog_mfleappad(machine_config &config)
 	SOFTWARE_LIST(config, "cart_list2").set_original("leapfrog_ltleappad_cart");
 }
 
+void leapfrog_leappad_state::leapfrog_ltleappad(machine_config &config)
+{
+	I8032(config, m_maincpu, 96000000/10); // (which exact type is it?)
+	m_maincpu->set_addrmap(AS_PROGRAM, &leapfrog_leappad_state::prog_map);
+	m_maincpu->set_addrmap(AS_IO, &leapfrog_leappad_state::ext_map);
+
+	// screenless
+
+	GENERIC_CARTSLOT(config, m_cart, generic_plain_slot, "leapfrog_ltleappad_cart");
+	m_cart->set_width(GENERIC_ROM16_WIDTH);
+	m_cart->set_device_load(FUNC(leapfrog_leappad_state::cart_load));
+
+	// no Software List yet
```

Comment:
There's a softlist, but I hooked it to "leapfrog_mfleappad" instead because we were missing the LittleTouch BIOS at that time. Can you please delete that softlist from MiFrist and move it to LittleTouch?

Reply:
alright, done